zarr/storage.py

```diff
@@ -1104,7 +1105,10 @@ class FSStore(Store):
         as a missing key
     dimension_separator : {'.', '/'}, optional
         Separator placed between the dimensions of a chunk.
-    storage_options : passed to the fsspec implementation
+    fs : fsspec.spec.AbstractFileSystem, optional
+        An existing filesystem to use for the store.
+    storage_options : passed to the fsspec implementation. Cannot be used
```

Comment:
There are a couple of extra options that FSMap takes over what a FileSystem takes (check, create, exceptions) that could go here.

Reply:
Are these the only possible options - https://github.com/fsspec/filesystem_spec/blob/201555581a9f0c0de810efc372704db952397330/fsspec/mapping.py#L35 - `check=False, create=False, missing_exceptions=None`?

Reply:
As the current [test failure](https://github.com/zarr-developers/zarr-python/runs/5820026384?check_suite_focus=true#step:7:1634) shows, it is not supported by `LocalFileSystem`.